hat/assets/js/apps/Iaso/domains/storages/hooks/useGetOperationsTypes.ts

```diff
@@ -0,0 +1,55 @@
+// @ts-ignore
+import { useSafeIntl } from 'bluesquare-components';
+import { IntlFormatMessage } from '../../../types/intl';
+
+import { DropdownOptions } from '../../../types/utils';
+
+import MESSAGES from '../messages';
+
+type OperationType =
+    | 'WRITE_PROFILE'
+    | 'RESET'
+    | 'READ'
+    | 'WRITE_RECORD'
+    | 'CHANGE_STATUS';
+
+export const useGetOperationsTypes = (): Array<DropdownOptions<string>> => {
+    const { formatMessage }: { formatMessage: IntlFormatMessage } =
+        useSafeIntl();
+    return [
+        {
+            value: 'WRITE_PROFILE',
+            label: formatMessage(MESSAGES.writeProfile),
+        },
+        {
+            value: 'RESET',
+            label: formatMessage(MESSAGES.reset),
+        },
+        {
+            value: 'READ',
+            label: formatMessage(MESSAGES.read),
+        },
+        {
+            value: 'WRITE_RECORD',
+            label: formatMessage(MESSAGES.writeRecord),
+        },
+        {
+            value: 'CHANGE_STATUS',
+            label: formatMessage(MESSAGES.changeStatus),
+        },
+    ];
+};
+
+export const useGetOperationsTypesLabel = (): ((
+    // eslint-disable-next-line no-unused-vars
+    operationTypeKey: OperationType,
+) => string) => {
+    const operationTypes = useGetOperationsTypes();
```

Comment:
same as above, if  `useGetOpertaionTypes`doesn't refresh when locale changes, this one won't either

Reply:
I've had the problem before, that I wrote a hook, which was in fact a normal, stateless function.

Reply:
iop i agree, i will let it like that, because in the future we will probably get those infos from the back-end